### ISSUE TITLE
Sender_info missing in alerts messages

### ISSUE THREAD

**Issue by guiguem** [project8/dripline-python#8]
_From @guiguem on November 11, 2016 2:0_

The sensor_info field is empty, this field might be required for a intelligent Slack display.
```
dripline.core.Service [sensor_value.trap_coil_3_current_limit]     (From App [routing_key])
{
    "timestamp": "2016-11-11T01:03:53Z", 
    "payload": {
        "value_cal": 0.25, 
        "value_raw": 0.25
    }, 
    "sender_info": {
        "username": "project8", 
        "exe": "/home/project8/python_environments/latest_version/bin/dragonfly", 
        "package": "dripline", 
        "service_name": "", 
        "hostname": "claude", 
        "version": "wp2.0.0", 
        "commit": "ga81e673"
    }
}
```

_Copied from original issue: project8/dripline#165_

**Comment by guiguem** [comment]
Might actually not be a problem for slack, but we should put it for completeness...
Not a priority if not a problem for Slack.

**Comment by wcpettus** [comment]
Fixed in 3.5.7
